TypeCobol.Test/Parser/Programs/Cobol85/EmptyArray.Mix.txt

```diff
@@ -0,0 +1,292 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Pgm2358.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+
+      *********************************************
+      *A. Usage is declared on array
+      *********************************************
+
+       01 Group-A.
+Line 11[15,20] <27, Error, Syntax> - Syntax error : A group item cannot be empty.
```

Comment:
Not relevant to this issue ?

Reply:
My bad, I totally forgot about this message which is kind of the same test that this issue.
We don't need both tests. We can discuss Monday what to do exactly in this PR.

Reply:
Not relevant test cases removed.